### PR TITLE
Compose 1.4.7 and Kotlin 1.8.21

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt [hunk 1]
```diff
@@ -785,7 +785,7 @@ class PaparazziPluginTest {
     assertThat(resourceFileContents[4]).isEqualTo("intermediates/assets/debug")
     assertThat(resourceFileContents[5]).isEqualTo("app.cash.paparazzi.plugin.test")
     assertThat(resourceFileContents[6]).isEqualTo("src/main/res,src/debug/res")
-    assertThat(resourceFileContents[7]).isEqualTo("caches/transforms-3/f35252aecaceded79eacc505f95f2ea2/transformed/external/res")
+    assertThat(resourceFileContents[7]).isEqualTo("caches/transforms-3/f573c9111f4bdbcb90c4bb6ea00d55bd/transformed/external/res")
```
Comment (jrodbx) [inline paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt:788]: https://github.com/cashapp/paparazzi/issues/851

File: paparazzi/paparazzi/libs/androidx-activity-1.5.0.jar
Comment (jrodbx) [inline paparazzi/paparazzi/libs/androidx-activity-1.5.0.jar:1]: https://github.com/cashapp/paparazzi/issues/783

will attempt to tackle this by next kotlin/compose bump

**Review by saket — APPROVED** [review]

File: paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziViewOwners.kt [hunk 2]
```diff
@@ -25,7 +25,6 @@ internal class PaparazziSavedStateRegistryOwner(
 internal class PaparazziOnBackPressedDispatcherOwner(
   private val lifecycleOwner: LifecycleOwner
 ) : OnBackPressedDispatcherOwner, LifecycleOwner by lifecycleOwner {
-  override fun getOnBackPressedDispatcher(): OnBackPressedDispatcher {
-    return OnBackPressedDispatcher { /* Swallow all back-presses. */ }
-  }
+  override val onBackPressedDispatcher: OnBackPressedDispatcher
+    get() = OnBackPressedDispatcher { /* Swallow all back-presses. */ }
```
[the remark this answers is not in the document]
Comment (jrodbx) [reply paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziViewOwners.kt:29]: Sorry, I'm not sure I understand the question.  Could you rephrase?

[the remark this answers is not in the document]
Comment (saket) [reply paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziViewOwners.kt:29]: whoops sorry I misread this PR. I thought I was reviewing changes in cash app 😄.
